misk/src/main/kotlin/misk/metrics/backends/MetricsBackendConfig.kt

```diff
@@ -0,0 +1,9 @@
+package misk.metrics.backends
+
+import misk.config.Config
+import misk.metrics.backends.graphite.GraphiteBackendConfig
+import misk.metrics.backends.stackdriver.StackDriverBackendConfig
+
+data class MetricsBackendConfig(
```

Comment:
This only seems to be used in `MetricsConfig`, which isn't used. Should this be moved to a separate PR?

Reply:
Same comment as above